internal/common/function/function.go

```diff
@@ -219,6 +220,25 @@ func ValidateBslSpec(ctx context.Context, clientInstance client.Client, nonAdmin
 	} else if nonAdminBsl.Spec.BackupStorageLocationSpec.Credential.Name == constant.EmptyString || nonAdminBsl.Spec.BackupStorageLocationSpec.Credential.Key == constant.EmptyString {
 		return fmt.Errorf("NonAdminBackupStorageLocation spec.bslSpec.credential.name or spec.bslSpec.credential.key is not set")
 	}
+	bslSyncPeriodErrorMessage := "NABSL spec.backupStorageLocationSpec.backupSyncPeriod (%v) can not be greater or equal non admin backupSyncPeriod (%v)"
+	if nonAdminBsl.Spec.BackupStorageLocationSpec.BackupSyncPeriod != nil {
+		if appliedBackupSyncPeriod <= nonAdminBsl.Spec.BackupStorageLocationSpec.BackupSyncPeriod.Duration {
+			return fmt.Errorf(
+				bslSyncPeriodErrorMessage,
+				nonAdminBsl.Spec.BackupStorageLocationSpec.BackupSyncPeriod.Duration, appliedBackupSyncPeriod,
+			)
+		}
+	} else {
+		// TODO how to get user configured default backup sync period?
```

Comment:
this can give false positives

if admin set defaultBackupSyncPeriod though DPA velero args, we do not know what is the value being used

Reply:
check for velero container args and push those values to NAC deployment and then use them. How does that sound ?

Reply:
added velero args to function